queens-psyc-clinic/app
# [FE FIX] drop-down filters for measure, item, age

### **FE FIX - changes drop-down filters**

**Testing**
1. navigate to branch: `git checkout change-filters`
2. `cd` to the `public` folder
3. run `npm install`
4. run `npm start`
5. ensure on all of the admin pages, the new filters are shown: Measure, Item, Min Age, Max Age.
<br>

For reference checkout issue: #102
